File: Erik_Vincent/Assignment1/Assignment1/Model/QLS/AST/Style/Widget/Radio.cs
namespace Assignment1.Model.QLS.AST.Style.Widget
{
    public class Radio : YesNoWidget
    {
        public Radio(int lineNumber, string yes, string no) : base(lineNumber, yes, no) { }

        public override void Accept(IStyleVisitor visitor) => visitor.Visit(this);
    }
}

File: GunOli/src/main/java/QLS/AST/Widgets/Radio.java
package QLS.AST.Widgets;

import QLS.Analysis.WidgetVisitorInterface;

import java.util.ArrayList;

public class Radio extends Widget {

    private ArrayList<String> values;
    public Radio(ArrayList<String> values, int line) {
        super(WidgetType.Radio, line);
        this.values = values;
    }

    @Override
    public <T> T accept(WidgetVisitorInterface<T> visitor) {
        return visitor.visit(this);
    }
}
